src/Sarif.Sarifer/SpamRule.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Microsoft.CodeAnalysis.Sarif.Sarifer
 {
-    internal class SpamRule
+    public class SpamRule
```

Comment:
>public [](start = 4, length = 6)

1. Revert to `internal`.
2. Suppress CA1812 ("Internal class not instantiated") in source.
3. Add a comment explaining that this class actually _is_ instantiated by JSON deserialization.